template/src/main/twirl/ch.epfl.scala.index.views/project/install.scala.html

```diff
@@ -26,27 +26,33 @@
   </ul>
   <div class="tab-content">
     <div role="tabpanel" class="tab-pane active" id="sbt">
-      <pre>@release.sbtInstall</pre>
+      <pre id="copy-sbt">@release.sbtInstall</pre>
+      <button class="btn-copy" data-clipboard-target="#copy-sbt">Copy</button>
```

Comment:
I suggest removing the leading `#` from the value of the `data-clipboard-target` attribute.